SAMv1.tex

```diff
@@ -40,6 +40,11 @@ \section{The SAM Format Specification}
 BAM file may optionally specify the version being used via the
 {\tt @HD VN} tag. For full version history see Appendix~\ref{sec:history}. 
 
+Unless explicitly specified elsewhere, all fields are encoded using 7-bit
+US-ASCII \footnote{Charset ANSI\_X3.4-1968 as defined in RFC1345.} in
+using the POSIX / C locale.  Regular expressions listed use the POSIX
+extended syntax.
```

Comment:
is there a reference for this "extended syntax"? I looked and found many dubious-looking sites...